Upstream changes with `xsimd` in master?

Did an `#include` get removed upstream that would cause this to error now?

I'm trying to use the current master branches of all 4 header library dependencies.

```
─  installing *source* package ‘xtensor’ ...
   Vendoring of xtensor headers is enabled.
   Cloning into 'xtl'...
   Cloning into 'xsimd'...
   Cloning into 'xtensor'...
   Cloning into 'xtensor-r'...
   > library('Rcpp');Rcpp::compileAttributes();
   > 
   > 
   clang++  -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.5/Resources/library/Rcpp/include" -I/usr/local/include  -I../inst/include --std=c++14 -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
   ** libs
   In file included from RcppExports.cpp:4:
   In file included from ./../inst/include/xtensor.h:4:
   ../inst/include/xtensor-r/roptional.hpp:90:34: error: use of undeclared identifier 'xsimd'
           using simd_return_type = xsimd::simd_return_type<xunderlying_type, requested_type>;
                                    ^
   1 error generated.
   make: *** [RcppExports.o] Error 1
   ERROR: compilation failed for package ‘xtensor’
```

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -23,7 +23,7 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda -c conda-forge
   # Host dependencies
-  - conda install xtensor=0.20.4 r-rcpp r-rinside r-devtools -c conda-forge
+  - conda install xtensor=0.20.5 r-rcpp r-rinside r-devtools -c conda-forge
   # Build dependencies 
   - conda install m2w64-gcc m2w64-make m2w64-toolchain m2-libbz2 posix -c conda-forge
   # Install miktex

--- a/.travis.yml
+++ b/.travis.yml
@@ -112,7 +112,7 @@ install:
     - conda config --set always_yes yes --set changeps1 no
     - conda update -q conda
     # Host dependencies
-    - conda install r r-rcpp xtensor==0.20.4 -c conda-forge
+    - conda install r r-rcpp xtensor==0.20.5 -c conda-forge
     # Build dependencies
     - conda install cmake -c conda-forge
     # Test dependencies

--- a/include/xtensor-r/roptional.hpp
+++ b/include/xtensor-r/roptional.hpp
@@ -87,7 +87,7 @@ namespace xt
         using underlying_type = std::decay_t<T>;
 
         template <class xunderlying_type, class requested_type>
-        using simd_return_type = xsimd::simd_return_type<xunderlying_type, requested_type>;
+        using simd_return_type = xt_simd::simd_return_type<xunderlying_type, requested_type>;
 
         using cpp_type = std::decay_t<T>;
         constexpr static int rtype = Rcpp::traits::r_sexptype_traits<cpp_type>::rtype;
@@ -120,7 +120,7 @@ namespace xt
         // auto proxy_simd_store(E& expr, std::size_t n, const simd& batch) const
         // {
         //     using simd_value_type = typename E::simd_value_type;
-        //     return expr.template store_simd<align>(n, xsimd::select(batch, simd_value_type(0), simd_value_type(NA_INTEGER)));
+        //     return expr.template store_simd<align>(n, xt_simd::select(batch, simd_value_type(0), simd_value_type(NA_INTEGER)));
         // }
     };
 

